icu4j/perf-tests/dateformatperf.pl

```diff
@@ -16,6 +16,9 @@
 # Test class
 my $TESTCLASS = 'com.ibm.icu.dev.test.perf.DateFormatPerformanceTest';
 
+my $CLASSES = './out/bin:../tools/misc/out/bin/:../icu4j.jar';
+
+
```

Comment:
nit: one empty line enough? Any rule or precedence that requires two lines?
Here and all other files.

Reply:
Removed extra line